src/plugins/intel_cpu/src/nodes/stft.cpp

```diff
@@ -73,15 +77,101 @@ bool STFT::created() const {
     return getType() == Type::STFT;
 }
 
+namespace {
+static void transpose_out4d(const uint8_t* in,
+                            uint8_t* out,
+                            const VectorDims& in_shape,
+                            const VectorDims& out_shape,
+                            size_t elem_size) {
+    const std::vector<size_t> axes_order{0, 2, 1, 3};
+    parallel_for3d(out_shape[0],
+                   out_shape[1],
+                   out_shape[2],
+                   [in, out, axes_order, &in_shape, &out_shape, elem_size](size_t i, size_t j, size_t k) {
+                       size_t in_indexes[4];
```

Comment:
Looks like we can work with `ijk` instead of static array `in_indexes`. But I don't insist 😊
However, the size of the array should be `3` anyway

Reply:
Right, changed the size to `3`.